src/App/component/AddProjectDialog.jsx

```diff
@@ -79,4 +80,4 @@ export default function AddProjectDialog({open, reloadProjects, handleClose}) {
       </DialogActions>
     </Dialog>
   )
-}
+}
```

Comment:
Suggest add a new line in the end of a file.
```suggestion
}

```